apps/css-workshop/pages/index.html

```diff
@@ -141,25 +131,25 @@ <h1>iTwinUI-CSS</h1>
     <img
       alt="GitHub last commit"
       src="https://img.shields.io/github/last-commit/itwin/iTwinUI"
+      draggable="false"
     >
     <a
       href="https://www.npmjs.com/package/@itwin/itwinui-css"
       rel="nofollow"
+      style="text-decoration: none;"
     >
       <img
         src="https://camo.githubusercontent.com/d372d1dae45885e50fb718a9c0a1a20544a3b0205842ca6cc4447545249c0e73/68747470733a2f2f696d672e736869656c64732e696f2f6e706d2f762f40697477696e2f697477696e75692d637373"
```

Comment:
Unrelated to the changes made in this PR, but looks like the npm image for `@itwin/itwinui-css` is broken.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/514893a1-c912-4284-bfb1-2a9a49e24c41)

Maybe this could also be fixed in this PR.

Reply:
I've updated to use badge from https://badge.fury.io/for/js/@itwin/itwinui-css